extensions/bigtable/bigtable_env.sh

```diff
@@ -56,3 +56,26 @@ GCE_SERVICE_ACCOUNT_SCOPES+=(
   'https://www.googleapis.com/auth/cloud-bigtable.data'
   'https://www.googleapis.com/auth/cloud-bigtable.data.readonly'
 )
+
+
+# hadoop common, hbase server, hbase client, hbase common, hadoop common, hbase annotation, bigtable hbase, alpn (ALPN_CLASSPATH), guava, 
+GCS_JARNAME=$(grep -o '[^/]*\.jar' <<< ${GCS_CONNECTOR_JAR})
+GCS_CLASSPATH=/home/hadoop/hadoop-install/share/hadoop/common/lib/${GCS_JARNAME}
+HADOOP_COMMON_CLASSPATH=/home/hadoop/hadoop-install/share/hadoop/common/hadoop-common-2.6.0.jar
+HBASE_SERVER_CLASSPATH=/home/hadoop/hbase-install/lib/hbase-server-1.0.1.jar
+HBASE_CLIENT_CLASSPATH=/home/hadoop/hbase-install/lib/hbase-client-1.0.1.jar
+HBASE_COMMON_CLASSPATH=/home/hadoop/hbase-install/lib/hbase-common-1.0.1.jar
+HBASE_ANNOTATIONS_CLASSPATH=/home/hadoop/hbase-install/lib/hbase-annotations-1.0.1.jar
+GUAVA_CLASSPATH=/home/hadoop/hadoop-install/share/hadoop/tools/lib/guava-11.0.2.jar
+ALPN_JAVA_OPTS=" -Xbootclasspath/p:${ALPN_CLASSPATH}"
+
```

Comment:
Sorry - lost track of where this was. 

Instead of having these here, you'll use the hbase shell script that's installed as part of install_bigtable_hbase. Further, it won't be on the path then either, but you can do something like RAW_CLASSPATH="$(${HBASE_INSTALL}/bin/hbase classpath)"